idaes/ui/tests/test_flowsheet.py

```diff
@@ -142,7 +143,7 @@ def flash_flowsheet():
     m.fs.flash = Flash(default={"property_package": m.fs.properties})
     m.fs.flash.inlet.flow_mol.fix(1)
     m.fs.flash.inlet.temperature.fix(368)
-    m.fs.flash.inlet.pressure.fix(101325)
+    m.fs.flash.inlet.pressure.fix(np.nan)
     m.fs.flash.inlet.mole_frac_comp[0, "benzene"].fix(0.5)
```

Comment:
set one to np.inf and another to -np.inf to check all three

Reply:
As a general question - what is the specific case why this is needed? I am fairly certain Pyomo does not support `numpy` floats (at least not yet), hence I wonder if there would ever be a case where a model output would be involve a numpy object.

Reply:
I'm not sure the exact situation but during our user testing one of the models that a user used ended up with a NaN in the stream contents.

If you want the specific model, I can ask the user if they would be willing to share the model via email.

Reply:
We might want to look into that, as there should be no way to get an NaN in a valid model anyway (solvers don't like those). This is sounding more like an issue that should be handled at the model level than in the UI.

Reply:
To capture what Andrew and I discussed:
The NaNs are a result of pandas filling in empty values with NaNs.
Andrew's PR #298 is going to take care of the empty values and replace them with "-" 
This PR still stands because we should future proof this in case we do support NaNs, Inf, and -Inf in the future